components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/BrandingPreferenceManagerImpl.java

```diff
@@ -547,6 +580,26 @@ private String getResourceNameForCustomText(String screen, String locale) {
         return StringUtils.upperCase(screen) + RESOURCE_NAME_SEPARATOR + StringUtils.lowerCase(formattedLocale);
     }
 
+    /**
+     * Return resource type for the given branding type.
+     *
+     * @param type Branding Type(Organization branding/Application branding).
+     * @return resource type of the branding resource.
+     */
+    private String getResourceType(String type) {
+
+        if (APPLICATION_TYPE.equals(type)) {
+            return APPLICATION_BRANDING_RESOURCE_TYPE;
+        }
+        return BRANDING_RESOURCE_TYPE;
+    }
+
+    private boolean isApplicationExists(String applicationName, String tenantDomain) {
+
+        //TODO: Implement this method to check the existence of application before add application branding preferences.
```

Comment:
Link the issue for this

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/commit/cf770738c9203eb8c32f84bd8258e30b6acf198a